src-care/topkg_care.mli

```diff
@@ -486,6 +486,7 @@ module Delegate : sig
   (** {2 Helpers} *)
 
   val publish_in_git_branch :
+    ?remote:string ->
```

Comment:
And the note should also be updated `is then pushed to the [remote] repository.`